plugins/util.go

```diff
@@ -84,6 +84,12 @@ func BigQueryURN(projectID, datasetID, tableID string) string {
 	return models.NewURN("bigquery", projectID, "table", fqn)
 }
 
+//urn:maxcompute:project:schema:table
+func MaxComputeURN(projectName, schemaName, tableName string) string {
+	fqn := fmt.Sprintf("%s:%s.%s", projectName, schemaName, tableName)
```

Comment:
sorry, I missed this one in RFC. should be using dots for all: `%s.%s.%s`

Reply:
addressed mas